admin/src/utils/vanillaInjection.js

```
@@ -0,0 +1,2381 @@
+          realPanel.style.boxShadow = '0 1px 4px rgba(33, 33, 52, 0.1)';
+
+          // Build panel based on data
+          let panelContent = `
+          <div style="display: flex; justify-content: space-between; align-items: center; margin-bottom: 16px;">
+            <h3 style="margin: 0; font-size: 16px; font-weight: 600; color: #4945ff;">Paper Trail</h3>
+        `;
+
+          if (trailData) {
+            panelContent += `<span style="font-size: 12px; color: #666687;">Version ${trailData.version}</span>`;
+          } else {
+            panelContent += `<span style="font-size: 12px; color: #666687;">No versions yet</span>`;
```

ESLint

[error] Strings must use singlequote.

(quotes)